### PR TITLE
Tc update

### PR DESCRIPTION
**Pull request by lightsofapollo**

### CODE REVIEW

File: _posts/2015-02-23-taskcluster-update.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Taskcluster is running by default on
+
+  - [try](https://treeherder.allizom.org/#/jobs?repo=try)
+  - [b2g-inbound](https://treeherder.allizom.org/#/jobs?repo=b2g-inbound)
+  - [mozilla-inbound](https://treeherder.allizom.org/#/jobs?repo=mozilla-inbound)
+  - [fx-team](https://treeherder.allizom.org/#/jobs?repo=fx-team)
+  - [mozilla-central](https://treeherder.allizom.org/#/jobs?repo=mozilla-central)
+
+
+You will see both buildbot run and taskcluster run jobs the "TC" jobs are
```
Comment (gregarndt) [inline _posts/2015-02-23-taskcluster-update.md]: Perhaps:
"In Treeherder you will see jobs run by both buildbot and taskcluster. The "TC" jobs are [...]"

File: _posts/2015-02-23-taskcluster-update.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+prefixed accordingly so you can tell the difference.
+
+This is the last big step to enabling TC as the default CI for many mozilla
+projects also new branches are a simple config away.
```
Comment (gregarndt) [inline _posts/2015-02-23-taskcluster-update.md]: "[...] projects.  Also, new branches are a simple config away."

File: _posts/2015-02-23-taskcluster-update.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Why is this a great thing? Just about everything is [in](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/mach_commands.py) [the](https://dxr.mozilla.org/mozilla-central/source/testing/docker) [tree](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/builds).
+
+This means you can easily add new builds/tests and immediately push them
+to try for testing (see the configs for [try](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/branches/try/job_flags.yml)
+
+Results currently show up only on [staging treeherder](https://treeherder.allizom.org)
+aside from the ease of adding tests there are a few other key benefits
```
Comment (gregarndt) [inline _posts/2015-02-23-taskcluster-update.md]: "Results currently show up only on [staging treeherder](https://treeherder.allizom.org).  Aside from [...]"

File: _posts/2015-02-23-taskcluster-update.md (hunk 2 above)
Comment (nullaus) [inline _posts/2015-02-23-taskcluster-update.md]: I would leave out the 'Also' and say that 'Adding new and existing branches is easily achieved with basic config changes.'

File: _posts/2015-02-23-taskcluster-update.md (hunk 3 above)
Comment (nullaus) [inline _posts/2015-02-23-taskcluster-update.md]: 'Results are shown on [...] only.' I would create a new paragraph that sets up the list of key benefits and retool the sentence -- 'Adding new tests is easier than ever but the improvements don't stop there. Other key benefits on linux include:'

File: _posts/2015-02-23-taskcluster-update.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+```sh
+# Pull tester image
+docker pull quay.io/mozilla/tester:0.0.14
+# Run tester image shell
+docker run -it quay.io/mozilla/tester:0.0.14 /bin/bash
+# <copy/paste stuff from task defintions into this>
+```
+
+#### Generally tests and builds are faster
```
Comment (nullaus) [inline _posts/2015-02-23-taskcluster-update.md]: 'Faster builds and test runs.' -- would drop the 'generally'.

File: _posts/2015-02-23-taskcluster-update.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Through this entire process we have been optimizing away overhead and using faster
+machines which means both build (and particularly test) times are faster.
+
+(Wins look [big](https://treeherder.allizom.org/#/jobs?repo=b2g-inbound&revision=233af1dfa476) but more in future blog post)
+
+#### What's missing ?
+
+There are a few missing pieces:
```
Comment (nullaus) [inline _posts/2015-02-23-taskcluster-update.md]: We can axe this sentence and simply jump into the list of things we are missing. :)

File: _posts/2015-02-23-taskcluster-update.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+This means you can easily add new builds/tests and immediately push them
+to try for testing (see the configs for [try](https://dxr.mozilla.org/mozilla-central/source/testing/taskcluster/tasks/branches/try/job_flags.yml)
+
+Results currently show up only on [staging treeherder](https://treeherder.allizom.org)
+aside from the ease of adding tests there are a few other key benefits
+on linux:
+
+#### We use [docker](https://dxr.mozilla.org/mozilla-central/source/testing/docker)
+
+Which means a completely identical CI environment is a docker pull away.
```
Comment (nullaus) [inline _posts/2015-02-23-taskcluster-update.md]: 'Docker enables easy cloning of CI environments.'

**Pull request merged**